multihost_test/Bugzillas/test_Regression.py

```diff
@@ -0,0 +1,118 @@
+""" Automation of Shadow Utils tests """
+
+from __future__ import print_function
+import pytest
+import subprocess
+import os
+import time
+from sssd.testlib.common.utils import SSHClient, sssdTools
+
+
+def execute_cmd(multihost, command):
+    """ Run command on client machine """
+    cmd = multihost.client[0].run_command(command)
+    return cmd
+
+
+class TestShadowUtilsRegression():
+    """
+    Automation of Shadow Utils tests
+    """
+    @pytest.mark.tier1
+    def test_bz507706(self, multihost, create_backup):
+        """
+        :title:bz507706 grpconv wants to take all memory if /etc/group contains duplicate entries
+        :id: dfe8883e-6728-11ee-b0c7-845cf3eff344
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=507706
+        :steps:
+          1. Add 10 user groups named group_0 to group_9.
+          2. Checks the number of groups with the name pattern group_ in the /etc/group file.
+            It asserts that there should be 10 such groups.
+          3. The content of the /etc/group file is saved into /tmp/anuj.
+          4. The groups with the name pattern group_ from /tmp/anuj are appended back into the /etc/group file.
+            This effectively doubles the number of such groups.
+          5. Checks again and asserts that the number of groups with the name pattern group_ in
+            the /etc/group file should now be 20.
+          6. The grpconv command is executed in the background (indicated by &).
+            This command initializes the shadow file.
+          7. Checks if the grpconv command is still running. If it is not running,
+            a subprocess.CalledProcessError exception will be raised.
+          8. Kill the grpconv process, if it's still running. Again, if the process doesn't exist,
+            a subprocess.CalledProcessError exception will be raised.
```

Comment:
Do you know why we need to do this?

Reply:
Just a double check , so that the process is not running

Reply:
And why we need to check? Wouldn't it be better to call `grpconv` synchronously and wait until it finishes?

Reply:
ok, i have removed this part .

Reply:
With this comment I meant that step 6 should run `grpconv` in the foreground, then steps 7 and 8 wouldn't be necessary

Reply:
changes are done

Reply:
I don't think we are understanding each other, so I added a new comment inline.